pkg/k8s/resource_test.go

```diff
@@ -41,3 +42,14 @@ func TestQKind(t *testing.T) {
 		}(testcase))
 	}
 }
+
+func TestDocker(t *testing.T) {
+	if _, err := exec.LookPath("docker"); err != nil {
+		t.Skip(err)
+	}
```

Comment:
NB: I added this, because without k3s, anything using `{{image}}` won't work on macOS CI.